packages/cli/src/commands/org/config/pull.ts

```diff
@@ -79,15 +87,19 @@ export default class Pull extends Command {
 
           Once the snapshot is created, you can pull it with the following command:
 
-            ${blueBright`coveo org:config:pull ${snapshot.id}`}
+            ${blueBright`coveo org:config:pull -s ${snapshot.id}`}
 
             `
       );
     }
   }
 
   private async refreshProject(snapshot: Snapshot) {
+    const {flags} = this.parse(Pull);
     const project = new Project(this.projectPath);
+    if (flags.git && !project.contains('.git')) {
+      exec(`git init ${this.projectPath}`);
```

Comment:
- Don't use non-synchronous methods from `child_process`. It's a mess to handle.
- Avoid to use `exec` to avoid [CWE-78](https://cwe.mitre.org/data/definitions/78.html) and [CWE-88](https://cwe.mitre.org/data/definitions/88.html). See CodeQL report for more information and remediation.
- Do use the existing utils for process Management, like `spawnSync`